server/models/Expense.js

```diff
@@ -83,7 +84,7 @@ export default function(Sequelize, DataTypes) {
       status: {
         type: DataTypes.STRING,
         defaultValue: status.PENDING,
-        allowNull: false,
+        allowNull: true,
```

Comment:
This won't work because there is also a `NOT NULL` constraint in the DB for this field, DB and model should be in sync. However I don't understand why we need this?

Reply:
Woops, that's just a total mistake and should not be there. I must have edited the wrong property. Good catch!